src/main.js

```diff
@@ -107,22 +128,68 @@ var savedPosters = [];
 var currentPoster;
 
 // event listeners go here 👇
-showRandomPoster.addEventListener('click', randomPosterButton);
+showRandomButton.addEventListener('click', randomPosterButton);
+// make a poster button
+makePosterButton.addEventListener('click',hideMainShowForm);
+// saved posters sections
+savedPostersButton.addEventListener('click', showSaved);
+// go back to main buttons
+nevermindButton.addEventListener('click', backToMainPage);
+backToMainButton.addEventListener('click', backToMainPage);
+
+
 
 
 // functions and event handlers go here 👇
+
+
 // (we've provided one for you to get you started)!
 function getRandomIndex(array) {
+  // return Math.floor(Math.random() * array.length);
   return array[Math.floor(Math.random() * array.length)];
+ }
+
+
+ function randomPosterButton() {
+   imageInput.innerText = getRandomIndex(images);
+   imageInput.src = getRandomIndex(images);
+   titleInput.innerText = getRandomIndex(titles);
+   quotesInput.innerText = getRandomIndex(quotes);
+ }
+function mainPage() {
+ imageInput.innerText = getRandomIndex(images);
+ imageInput.src = getRandomIndex(images);
+ titleInput.innerText = getRandomIndex(titles);
+ quotesInput.innerText = getRandomIndex(quotes);
 }
-function randomPosterButton() {
-  imageInput.innerText = getRandomIndex(images);
-  imageInput.src = getRandomIndex(images);
-  titleInput.innerText = getRandomIndex(titles);
-  quotesInput.innerText = getRandomIndex(quotes);
+mainPage();
+
+// hidden functions
+ function hideMainPoster() {
+   mainPoster.classList.add('hidden');
+ }
+
+function showForm (){
+  makePosterForm.classList.remove('hidden');
 }
 
-imageInput.innerText = getRandomIndex(images);
-imageInput.src = getRandomIndex(images);
-titleInput.innerText = getRandomIndex(titles);
-quotesInput.innerText = getRandomIndex(quotes);
+function showSavedPosters() {
+  savedPostersPage.classList.remove('hidden');
+}
+
```

Comment:
These are good, but if you wanted to you could write a reusable function. When I write out functions like this and see they are very similar I like to refactor them into one. There is a parameter (or two) you can use to make a very dynamic function. You could even use that function inside other functions